frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/playgroundResultStrategyLists.tsx

```diff
@@ -67,24 +67,40 @@ export const PlaygroundResultStrategyLists = ({
 );
 
 interface IWrappedPlaygroundResultStrategyListProps {
-    strategies: PlaygroundFeatureSchemaStrategies;
+    feature: PlaygroundFeatureSchema;
     input?: PlaygroundRequestSchema;
 }
 
+const resolveHintText = (feature: PlaygroundFeatureSchema) => {
+    if (
+        feature.hasUnsatisfiedParent &&
+        !feature.isEnabledInCurrentEnvironment
+    ) {
+        return 'If environment was enabled and parent dependencies were satisfied';
+    }
+    if (feature.hasUnsatisfiedParent) {
+        return 'If parent dependencies were satisfied';
+    }
+    if (!feature.isEnabledInCurrentEnvironment) {
+        return 'If environment was enabled';
+    }
+    return '';
```

Comment:
should not occur, will take care of it with the next PR with extensive test suite